packages/services/src/serverconnection.ts

```diff
@@ -93,14 +93,14 @@ namespace ServerConnection {
     dataType?: string;
 
     /**
-     * The outgoing content type, used to set the `Content-Type` header.  Defaults to `'application/json'` when there is sent data.
+     * The outgoing content type, used to set the `Content-Type` header.
      */
     contentType?: string;
 
     /**
      * The request data.
      */
-    data?: any;
+    data?: Blob | BufferSource | FormData | URLSearchParams | ReadableStream | string;
```

Comment:
Sorry, things are very hectic, so I haven't had a chance to check this yet.

I'm confused about line 352 of this file - why, if there's no data, is the Content-Type set to be `application/json`?